src/SSPL-1.0.xml

```diff
@@ -0,0 +1,643 @@
+          License. If you cannot convey a covered work so as to satisfy
+          simultaneously your obligations under this License and any other
+          pertinent obligations, then as a consequence you may
+                        <br></br>
+          not convey it at all. For example, if you agree to terms
+          that obligate you to collect a royalty for further conveying
+          from those to whom you convey the Program, the only
+          way you could satisfy both those terms and this License
+          would be to refrain entirely from conveying the Program.
+                    </p>
+                </item>
+                <item>
+                    <bullet>13.</bullet>Offering the Program as a Service.
+                    <p>
+If you make the functionality of the Program or a modified version available to third parties as a service, you must make the Service Source Code available via network download to everyone at no charge, under the terms of this License. Making the functionality of the Program or modified version available to third parties as a service includes, without limitation, enabling third parties to interact with the functionality of the Program or modified version remotely through a computer network, offering a service the value of which entirely or primarily derives from the value of the Program or modified version, or offering a service that accomplishes for users the primary purpose of the Program or modified version.</p>
+                    <p>
+&#8220;Service Source Code&#8221; means the Corresponding Source for the Program or the modified version, and the Corresponding Source for all programs that you use to make the Program or modified version available as a service, including, without limitation, management software, user interfaces, application program interfaces, automation software, monitoring software, backup software, storage software and hosting software, all such that a user could run an instance of the service using the Service Source Code you make available.</p>
+                </item>
+                <item>
+                    <bullet>14.</bullet>Revised Versions of this License.
+                    <p>
+MongoDB, Inc. may publish revised and/or new versions of the Server Side Public License from time to time. Such new versions will be similar in spirit to the present version, but may differ in detail to address new problems or concerns.</p>
+                    <p>
+Each version is given a distinguishing version number. If the Program specifies that a certain numbered version of the Server Side Public License &#8220;or any later version&#8221; applies to it, you have the option of following the terms and conditions either of that numbered version or of any later version published by MongoDB, Inc. If the Program does not specify a version number of the Server Side Public License, you may choose any version ever published by MongoDB, Inc.</p>
```

Comment:
Same comment as line 587 above re: these numbered tags vs. regular quote marks that are used elsewhere in the license.